flash

```diff
@@ -141,7 +141,7 @@ fi
 
 case "${OSTYPE}" in
   darwin*)
-    alias grep="grep --color=never"
+    grep() { grep --color=never "$@"; }
```

Comment:
Fixed shellcheck error

```
In flash line 144:
    alias grep="grep --color=never"
          ^-----------------------^ SC2262: This alias can't be defined and used in the same parsing unit. Use a function instead.


In flash line 162:
        echo "$_RET" | grep -v "from "
                       ^-------------^ SC2263: Since they're in the same parsing unit, this command will not refer to the previously mentioned alias.
```